### ISSUE TITLE
Obsolete getAuthor method on BaseElement?

### ISSUE THREAD

**Issue by michalkleiner** [dnadesign/silverstripe-elemental#663]
Is the method `getAuthor` (https://github.com/dnadesign/silverstripe-elemental/blob/master/src/Models/BaseElement.php#L921) used anywhere? There's no has_one for it so it's always null.

**Comment by ScopeyNZ** [comment]
Good point. Looks like it's some hangover from versioning? I can't see a usage - interested to see what it returns though (if anything). Makes sense to deprecate at least imo.
